src/frontend/src/lib/components/hero/Balance.svelte

```diff
@@ -13,7 +13,7 @@
 
 <span class="flex flex-col gap-2">
 	<output
-		class={`break-words ${(token?.balance?.toBigInt() ?? 0n) === 0n ? 'opacity-50' : 'opacity-100'} inline-flex w-full flex-row justify-center gap-3 text-4xl font-bold lg:text-5xl`}
+		class={`inline-flex w-full flex-row justify-center gap-3 break-words text-4xl font-bold lg:text-5xl`}
```

Comment:
since it is not required that is a variable anymore, can you please remove the brackets?